todo-app/src/index.js

```diff
@@ -1,5 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import Todos from './component/todos';
+import Header from "./component/header";
+import JSON from "./todo.json";
```

Comment:
I'd like to see you name your data differently here. Calling _anything_ `JSON` is going to lead to serious confusion down the line, as `JSON` is a built-in with native methods in JavaScript, eg `JSON.stringify()`. I'm actually pretty surprised you were able to assign a variable to this keyword, never seen this before.